src/main/java/seedu/recruittrackpro/logic/commands/FindCommand.java

```diff
@@ -18,7 +18,7 @@ public class FindCommand extends Command {
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose names contain any of "
             + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
-            + "Parameters: " + PREFIX_NAME + "KEYWORD [MORE_KEYWORDS]...\n"
```

Comment:
Not a big issue, but do you think adding a short description that elaborates the purpose of `--contain-all` will be helpful?
```suggestion
            + "If the optional `--contain-all` flag is provided, only persons whose names contain all specified "
            + "keywords (in any order) will be listed.\n\n"